R/defaults.R

```diff
@@ -1,23 +1,35 @@
 ## vim:textwidth=128:expandtab:shiftwidth=4:softtabstop=4
 
-#' Get the default Argo cache directory
+#' Get Default Values
```

Comment:
It might be more informative if this line were `Get Default Server Value`.  Maybe I'm missing something, though.  Also, I think the title of the function says it all, and so perhaps that's why you chose to shorten the title.

Reply:
Sorry, of course I meant 'cache directory', not 'server' here.

Reply:
Oh, I take it back.  You're documenting all three together.  All is good.  Sorry for the noise.